frontends/web/src/routes/account/receive/receive.tsx

```diff
@@ -280,11 +280,11 @@ export const Receive = ({
                       primary>
                       {t('receive.verifyBitBox02')}
                     </Button>
-                    <ButtonLink
+                    <Button
```

Comment:
Why don't you use the Backbutton component here and in other places?

Reply:
BackButton is just a simple wrapper so we can use the hook in class based component. 

In one place it shares the onClick function with useEsc and I guess we do that more in the future. So should the new BackButton do this as well? what if there is another popup that listens to esc etc?

I prefer to keep it simple for now, but no strong opinion.